package.json

```
@@ -1,111 +1,36 @@
-    "prepare": "husky",
-    "release": "release-it",
-    "changelog": "lerna-changelog"
-  },
-  "dependencies": {
-    "@babel/core": "^7.24.4",
-    "ember-cli-babel": "^8.2.0",
-    "ember-cli-htmlbars": "^6.3.0"
+    "build": "npm run build --workspace ember-liveagent",
+    "lint": "npm run lint --workspaces --if-present",
+    "lint:fix": "npm run lint:fix --workspaces --if-present",
+    "prepare": "npm run build",
+    "start": "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
+    "start:addon": "npm start --workspace ember-liveagent -- --no-watch.clearScreen",
+    "start:test-app": "npm start --workspace test-app",
+    "test": "npm run test --workspaces --if-present",
+    "test:ember": "npm run test:ember --workspaces --if-present"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.24.1",
-    "@babel/plugin-proposal-decorators": "^7.24.1",
-    "@ember/optional-features": "^2.1.0",
-    "@ember/test-helpers": "^3.3.0",
-    "@embroider/test-setup": "^4.0.0",
-    "@glimmer/component": "^1.1.2",
-    "@glimmer/tracking": "^1.1.2",
-    "@release-it-plugins/lerna-changelog": "^6.1.0",
-    "broccoli-asset-rev": "^3.0.0",
-    "concurrently": "^8.2.2",
-    "ember-auto-import": "^2.7.2",
-    "ember-cli": "~5.8.0",
-    "ember-cli-clean-css": "^3.0.0",
-    "ember-cli-code-coverage": "^3.0.0",
-    "ember-cli-dependency-checker": "^3.3.2",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2",
-    "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^8.2.3",
-    "ember-qunit": "^8.0.2",
-    "ember-resolver": "^11.0.1",
-    "ember-source": "~5.8.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^6.0.0",
-    "ember-try": "^3.0.0",
-    "eslint": "^8.57.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-ember": "^12.0.2",
-    "eslint-plugin-n": "^17.2.1",
-    "eslint-plugin-prettier": "^5.1.3",
-    "eslint-plugin-qunit": "^8.1.1",
-    "husky": "^9.0.11",
-    "lerna-changelog": "^2.2.0",
-    "lint-staged": "^15.2.2",
-    "loader.js": "^4.7.0",
-    "prettier": "^3.2.5",
-    "qunit": "^2.20.1",
-    "qunit-dom": "^3.1.1",
-    "release-it": "^17.2.0",
-    "stylelint": "^16.3.1",
-    "stylelint-config-standard": "^36.0.0",
-    "stylelint-prettier": "^5.0.0",
-    "webpack": "^5.91.0"
-  },
-  "peerDependencies": {
-    "ember-source": "^4.0.0"
-  },
-  "engines": {
-    "node": ">= 18"
-  },
-  "ember": {
-    "edition": "octane"
-  },
-  "ember-addon": {
-    "configPath": "tests/dummy/config"
-  },
+    "@glint/core": "^1.2.1",
+    "concurrently": "^8.2.0",
+    "prettier": "^3.0.3",
+    "prettier-plugin-ember-template-tag": "^1.1.0"
+  },
+  "workspaces": [
+    "ember-liveagent",
+    "test-app"
+  ],
   "volta": {
     "node": "18.20.0",
     "npm": "10.5.0"
   },
   "lint-staged": {
-    "*.js": "eslint --cache --fix",
+    "*.ts": "eslint --cache --fix",
```

```suggestion
    "*.{js,ts,cjs,mjs}": "eslint --cache --fix",
```

But it has no effect for now due the missing lint-staged/husky, hence can be ✂️ completely until restored back.